### ISSUE TITLE
NVDA is unable to read several pieces of information in File Zilla

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1473]
**Reported by elliott94 on 2011-04-26 17:54**
When using the File Zilla FTP client with NVDA, information such as the remaining amount of data to be transfered, current download/upload speed, and current time remaining is unable to be unread. The current data remaining to be transfered is visible by looking at the Status Bar, however in most cases this info doesn't refresh when using the "Read Status Bar" command.

**Comment by bhavyashah** [comment]
Based on https://github.com/nvaccess/nvda/issues/1473#issuecomment-155289633 and https://github.com/nvaccess/nvda/issues/1473#issuecomment-155289634, it seems that this ticket can be closed as worksforme. @ehollig

**Comment by ehollig** [comment]
Sounds like everything is sorted here, and others have reported that it works. Closing as worksforme
